src/content/fr/2019/markup.md

```diff
@@ -0,0 +1,283 @@
+  <figcaption id="fig2-caption">Figure 2. Distribution de l'analyse des fréquences d'élément effectuée par Hixie en 2005.</figcaption>
+</figure>
+
+<figure id="fig3">
+  <a href="/static/images/2019/03_Markup/fig3.png">
+    <img src="/static/images/2019/03_Markup/fig3.png" alt="Figure 3. Fréquences des éléments à partir de 2019" aria-labelledby="fig3-caption" aria-describedby="fig3-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbHgqcSepZye6DrCTpifFAUYxKT1hEO56585awyMips8oiPMLYu20GETuIE8mALkm814ObJyktEe2P/pubchart?oid=2141583176&amp;format=interactive">
+  </a>
+  <div id="fig3-description" class="visually-hidden">Graphique représentant environ 2 500 pages commençant par environ 30 éléments, il passe à 6 876 pages et compte 283 éléments, avant de se terminer d'une manière relativement linéaire jusqu'à 327 pages comportant 2,00 éléments.</div>
+  <figcaption id="fig3-caption">Figure 3. Fréquences des éléments à partir de 2019.</figcaption>
+</figure>
+
+En comparant les dernières données de la figure 3 à celles du rapport d'Hixie de 2005 à la figure 2, nous pouvons constater que la taille moyenne des arbres DOM a augmenté.
+
+<figure id="fig4">
+  <img src="/static/images/2019/03_Markup/hixie_element_types_per_page.png" alt="Histogramme de l'analyse effectuée par Hixie en 2005 sur les types d'éléments par page" aria-labelledby="fig4-caption" aria-describedby="fig4-description" width="600">
+   <div id="fig4-description" class="visually-hidden">Cette graphe de fréquence relative par une courbe en cloche autour du point des 19 éléments..</div>
+  <figcaption id="fig4-caption">Figure 4. Histogramme de l'analyse effectuée par Hixie en 2005 sur les types d'éléments par page.</figcaption>
+</figure>
+
+<figure id="fig5">
+    <a href="/static/images/2019/03_Markup/fig5.png">
+      <img src="/static/images/2019/03_Markup/fig5.png" alt="Figure 5. Histogramme des types d'éléments par page à partir de 2019." aria-labelledby="fig5-caption" aria-describedby="fig5-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbHgqcSepZye6DrCTpifFAUYxKT1hEO56585awyMips8oiPMLYu20GETuIE8mALkm814ObJyktEe2P/pubchart?oid=1500675289&amp;format=interactive">
+    </a>
+    <div id="fig5-description" class="visually-hidden">Ce graphique représente le nombre moyen d'éléments est une courbe en cloche autour des 30 éléments marqués, telle qu'utilisée par 308 168 milliers de sites.</div>
+  <figcaption id="fig5-caption">Figure 5. Histogramme des types d'éléments par page à partir de 2019.</figcaption>
+</figure>
+
+Nous pouvons voir sur les deux que nombre moyen de types d'éléments par page a augmentés, ainsi que le nombre maximal d'éléments uniques que nous rencontrons.
```

Comment:
its good to write : " d'éléments par page ont augmenté " instead of "d'éléments par page a augmentés" car ils sont "des éléments " et non-pas un seul "élément"

Reply:
je n'ai as payé mon attention au verbe car il ne s'accorde pas en genre et en nombre